packages/core/saved-objects/core-saved-objects-browser/jest.config.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
+  preset: '@kbn/test/jest_node',
```

Comment:
this is supposed to work on browser, I believe the preset is `@kbn/test` instead of `jest_node`

Reply:
If the tests don't require JSDom the node preset is still more efficient and has less hackery to deal with, but the impact is usually pretty mild either way.